src/app/user-profile/pages/edit-profile-page/edit-profile-page.component.ts

```diff
@@ -0,0 +1,97 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Store } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { take } from 'rxjs/operators';
+import { UserApiService } from './../../services/user-api.service';
+import { UserService } from 'src/app/shared/services/user.service';
+import { selectUserLogin, selectUserName } from 'src/app/redux/selectors';
+import * as UserActions from '../../../redux/actions/index';
+import { passwordStrengthValidator } from 'src/app/login/validators/password-strength.validator';
+
+@Component({
+  selector: 'app-edit-profile-page',
+  templateUrl: './edit-profile-page.component.html',
+  styleUrls: ['./edit-profile-page.component.scss'],
+})
+export class EditProfilePageComponent implements OnInit {
+
+  constructor(
+    private store: Store,
+    private api: UserApiService,
+    private userService: UserService,
+    private router: Router,
+  ) { }
+
+  ngOnInit(): void {
+    this.store.select(selectUserName).pipe(take(1)).subscribe({
+      next: (name) => this.nameValue = name,
+    });
+    this.store.select(selectUserLogin).pipe(take(1)).subscribe({
+      next: (login) => this.loginValue = login,
+    });
+
+    this.editProfileForm.get('name')!.setValue(this.nameValue);
+    this.editProfileForm.get('login')!.setValue(this.loginValue);
+  }
+
+  public nameValue: string = '';
```

Comment:
no need to write a type for primitive with initial value

Reply:
done